tests/phpunit/tests/link/getEditCommentLink.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @group link
+ * @group comment
+ * @covers ::get_next_comments_link
+ */
+class Tests_Link_GetEditCommentLink extends WP_UnitTestCase {
+
+	public static $comment_ids;
+	public static $user_ids;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$comment_ids = array(
+			'valid'   => $factory->comment->create( array( 'comment_content' => 'Test comment' ) ),
+			'invalid' => 12345, // Invalid comment ID
+		);
+
+		self::$user_ids = array(
+			'admin'      => $factory->user->create( array( 'role' => 'administrator' ) ),
+			'subscriber' => $factory->user->create( array( 'role' => 'subscriber' ) ),
+		);
+	}
+
+	public function set_up() {
+		parent::set_up();
+		wp_set_current_user( self::$user_ids['admin'] );
+	}
+
+	public function test_get_edit_comment_link_display_context() {
+		$comment_id   = self::$comment_ids['valid'];
+		$expected_url = admin_url( 'comment.php?action=editcomment&amp;c=' . $comment_id );
+		$actual_url   = get_edit_comment_link( $comment_id, 'display' );
+
+		$this->assertSame( $expected_url, $actual_url );
+	}
+
+	public function test_get_edit_comment_link_view_context() {
+		$comment_id   = self::$comment_ids['valid'];
+		$expected_url = admin_url( 'comment.php?action=editcomment&c=' . $comment_id );
+		$actual_url   = get_edit_comment_link( $comment_id, '' );
+
+		$this->assertSame( $expected_url, $actual_url );
+	}
+
+	public function test_get_edit_comment_link_invalid_comment() {
+		$comment_id         = self::$comment_ids['invalid'];
```

Comment:
Per my above comment, here you can simply hard-code the invalid ID, no need to store in a class property.
```suggestion
		$comment_id         = 12345;
```